### ISSUE TITLE
Vagrant provisioner fails to install mailcatcher

### ISSUE THREAD

**Issue by zarino** [mysociety/mapit.mysociety.org#30]
```
vagrant@vagrant-ubuntu-trusty-64:~$ sudo gem install --no-rdoc --no-ri mailcatcher
ERROR:  Error installing mailcatcher:
	mime-types-data requires Ruby version >= 2.0.
vagrant@vagrant-ubuntu-trusty-64:~$ ruby -v
ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
```

What’s confusing is, rubygems.org says that mailcatcher 0.6.5 only requires Ruby 1.9.3 or higher.

Since the Vagrantfile attempts to start mailcatcher whenever the VM starts, the lack of mailcatcher causes `vagrant up` to fail. A temporary fix is to comment out the `Start mailcatcher every time we start the VM` block at the end of the Vagrantfile.

**Comment by zarino** [comment]
Oh look! https://github.com/sj26/mailcatcher/issues/277

@dracos Shall I follow their suggestion and update the Vagrantfile to read…?

    sudo gem install mime-types --version "< 3"
    sudo gem install --no-rdoc --no-ri --conservative mailcatcher

**Comment by zarino** [comment]
Although, interestingly, that then fails too:

```
vagrant@vagrant-ubuntu-trusty-64:~$ sudo gem install --no-rdoc --no-ri --conservative mailcatcher
Fetching: mail-2.6.4.gem (100%)
Fetching: rack-1.6.5.gem (100%)
Fetching: tilt-2.0.5.gem (100%)
Fetching: rack-protection-1.5.3.gem (100%)
Fetching: sinatra-1.4.7.gem (100%)
Fetching: sqlite3-1.3.13.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing mailcatcher:
	ERROR: Failed to build gem native extension.

        /usr/bin/ruby1.9.1 extconf.rb
checking for sqlite3.h... no
sqlite3.h is missing. Try 'brew install sqlite3',
'yum install sqlite-devel' or 'apt-get install libsqlite3-dev'
and check your shared library search path (the
location where your sqlite3 shared library is located).
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of
necessary libraries and/or headers.  Check the mkmf.log file for more
details.  You may need configuration options.

Provided configuration options:
	--with-opt-dir
	--without-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/usr/bin/ruby1.9.1
	--with-sqlite3-config
	--without-sqlite3-config
	--with-pkg-config
	--without-pkg-config
	--with-sqlite3-dir
	--without-sqlite3-dir
	--with-sqlite3-include
	--without-sqlite3-include=${sqlite3-dir}/include
	--with-sqlite3-lib
	--without-sqlite3-lib=${sqlite3-dir}/lib

Gem files will remain installed in /var/lib/gems/1.9.1/gems/sqlite3-1.3.13 for inspection.
Results logged to /var/lib/gems/1.9.1/gems/sqlite3-1.3.13/ext/sqlite3/gem_make.out
```

**Comment by dracos** [comment]
Fixed in 14e9f390d97206f0a771d62ed409c2dd056ccdbd
